php/libraries/SinglePointLogin.class.inc

```diff
@@ -415,8 +415,11 @@ class SinglePointLogin
                     $user->updatePassword($password);
                 }
 
-                if ($row['Active'] == 'N') {
-                    $this->_lastError = "Your account has been deactivated."
+                if ($row['Active'] == 'N'
+                    || $this->disabledDueToInactivity($username, 365)
+                ) {
+                    $this->_lastError = "Your account has been deactivated,"
+                        . " possibly due to inactivity."
```

Comment:
Given the new function, I'd suggest separating this into 2 error messages (1 for each case) to avoid using the word "possibly"